packages/plugin-react-swc/package.json

```diff
@@ -1,15 +1,36 @@
 {
   "name": "@vitejs/plugin-react-swc",
   "version": "3.8.1",
+  "license": "MIT",
+  "author": "Arnaud Barré (https://github.com/ArnaudBarre)",
+  "description": "Speed up your Vite dev server with SWC",
+  "keywords": [
+    "vite",
+    "vite-plugin",
+    "react",
+    "swc",
+    "react-refresh",
+    "fast refresh"
+  ],
   "type": "module",
-  "private": true,
```

Comment:
Can we keep this one? This is to avoid running npm publish at the root

Reply:
Reverted this part 👍